src/main/java/com/company/BookingList.java

```diff
@@ -0,0 +1,118 @@
+package com.company;
+import com.company.model.Client;
+import com.company.model.Movie;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.company.usinglist.MovieList.createFirstMovieList;
+import static com.company.usinglist.MovieList.createSecondMovieList;
+
+public class BookingList {
```

Comment:
You have to refactor this method. It is a mess. More than 100 lines for a method!